datafusion/common/src/file_options/parquet_writer.rs

```diff
@@ -183,10 +135,87 @@ impl TryFrom<&TableParquetOptions> for ParquetWriterOptions {
             }
         }
 
-        // ParquetWriterOptions will have defaults for the remaining fields (e.g. sorting_columns)
-        Ok(ParquetWriterOptions {
-            writer_options: builder.build(),
-        })
+        Ok(builder)
+    }
+}
+
+impl ParquetOptions {
+    /// Convert the global session options, [`ParquetOptions`], into a single write action's [`WriterPropertiesBuilder`].
+    ///
+    /// The returned [`WriterPropertiesBuilder`] can then be further modified with additional options
+    /// applied per column; a customization which is not applicable for [`ParquetOptions`].
+    pub fn writer_props_from_global_opts(&self) -> Result<WriterPropertiesBuilder> {
```

Comment:
Perhaps we could call this `into_writer_properties_builder()` to keep the naming constent